### PR TITLE
Installation instructions should spell out packages on CentOS

### PR DESCRIPTION
**Pull request by manur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Once you've done some hacking and you'd like to have your work merged, you'll ne
 
 ## Prerequisites
 
-### Linux
+### Debian/Ubuntu
```
Comment (modeswitch) [inline README.md]: These package names aren't correct for Debian/Ubuntu. They're just the library names.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,17 @@ Once you've done some hacking and you'd like to have your work merged, you'll ne
 * libnss-devel
 * libexpat-devel
 
+### CentOS/Red Hat
```
Comment (modeswitch) [inline README.md]: Can you change this to a shell invocation that can be copy-pasted to install all the packages? Something like `yum install ...`.

**Comment by modeswitch** [comment]
Please add yourself to the list of contributors (if you like).

**Comment by manur** [comment]
Thanks for the suggestions, @modeswitch 

I've spelt out pre-requisite instructions as best as I could map them from the respective package managers in [apt-get, yum, brew].

File: README.md [hunk 3]
```diff
@@ -22,10 +20,29 @@ Once you've done some hacking and you'd like to have your work merged, you'll ne
 * libnss-devel
 * libexpat-devel
 
+### Debian/Ubuntu
+
+To install prerequisites with the apt-get package manager,
+
+`apt-get install python2.7 git-all pkg-config libncurses5-dev libssl-dev libnss3-dev libexpat-dev  `
+
+### CentOS/Red Hat
```
Comment (modeswitch) [inline README.md]: This covers Fedora also.

**Comment by modeswitch** [comment]
Merged manually.

**Pull request closed without merging**